src/types.ts

```diff
@@ -22,18 +21,30 @@ export interface PageScrollStateChangedEvent {
   pageScrollState: PageScrollState;
 }
 
-export interface PagerViewProps {
+export interface EagerPagerProps {
```

Comment:
I think, we should store all props in one interface. Could you merge `EagerPagerProps` and `PagerViewProps` together ?

Reply:
The problem with that, is it would require `data`, `keyExtractor`, and `renderItem` be passed to the `EagerPager`, which does not use them.  It would also mean `buffer` and `maxRenderWindow` are accepted props, but are in reality ignored by `EagerPager`.